<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WebBrowserConfig</class>
 <widget class="QWidget" name="WebBrowserConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Auto refresh:</string>
     </property>
     <property name="buddy">
      <cstring>autoRefresh</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QCheckBox" name="autoRefresh">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Interval:</string>
     </property>
     <property name="buddy">
      <cstring>autoRefreshInterval</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="intervalGroup" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="KIntSpinBox" name="autoRefreshInterval">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>0</height>
         </size>
        </property>
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Drag to scroll the page:</string>
     </property>
     <property name="buddy">
      <cstring>dragToScroll</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="dragToScroll">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>autoRefresh</sender>
   <signal>toggled(bool)</signal>
   <receiver>intervalGroup</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
